rfcs/0007-filtering.md

```diff
@@ -0,0 +1,241 @@
+# RFC Proposal
+[RFC Proposal]: #rfc-proposal
+
+* RFC Proposal ID: `0007-filtering-fields`
+* Start Date: 2020-08-21
+* RFC PR: [rfcs#0007](https://github.com/CMSgov/beneficiary-fhir-data/pull/345)
+* JIRA Ticket(s):
+    * [https://jira.cms.gov/browse/AB2D-1863](https://jira.cms.gov/browse/AB2D-1863)
+
+This proposal will suggest adding server side filtering to API calls that return EOBs to the client in order to 
+reduce on the amount of data that needs to be transferred. 
```

Comment:
We already do a lot of filtering, so let's reframe this summary to reflect that.
```suggestion
This proposal a new approach to server side filtering of API call results. The new approach is designed to be more comprehensive, and avoid all of the branching currently used for filtering in the transformers.
```